Remove BOOST_MAKE_PFTO_WRAPPER macros, which were long deprecated
and finally removed with boost-1.59.

diff -aur package.pristine/tools/displays/piqsl/piqsldisplay.cpp package.new/tools/displays/piqsl/piqsldisplay.cpp
--- package.pristine/tools/displays/piqsl/piqsldisplay.cpp	2015-12-12 12:52:50.455773602 +0100
+++ package.new/tools/displays/piqsl/piqsldisplay.cpp	2015-12-12 12:54:31.291661954 +0100
@@ -450,8 +450,8 @@
 
 	TiXmlElement* bucketDataXML = new TiXmlElement("BucketData");
 	std::stringstream base64Data;
-	std::copy(	base64_text(BOOST_MAKE_PFTO_WRAPPER(data)), 
-				base64_text(BOOST_MAKE_PFTO_WRAPPER(data + bufferlength)), 
+	std::copy(	base64_text(data), 
+				base64_text(data + bufferlength), 
 				std::ostream_iterator<char>(base64Data));
 	TiXmlText* dataTextXML = new TiXmlText(base64Data.str());
 	dataTextXML->SetCDATA(true);
diff -aur package.pristine/tools/piqsl/imagelistmodel.cpp package.new/tools/piqsl/imagelistmodel.cpp
--- package.pristine/tools/piqsl/imagelistmodel.cpp	2015-12-12 12:52:50.465773524 +0100
+++ package.new/tools/piqsl/imagelistmodel.cpp	2015-12-12 12:53:26.088831765 +0100
@@ -42,11 +42,6 @@
 #include <boost/archive/iterators/transform_width.hpp>
 #include <boost/archive/iterators/remove_whitespace.hpp>
 #include <boost/version.hpp>
-#if BOOST_VERSION < 103700
-#   include <boost/pfto.hpp>
-#else
-#   include <boost/serialization/pfto.hpp>
-#endif
 
 
 #include "displayserverimage.h"
@@ -286,7 +281,7 @@
                                 std::string data = dataText->Value();
                                 std::vector<unsigned char> binaryData;
                                 binaryData.reserve(count);
-                                base64_binary ti_begin = base64_binary(BOOST_MAKE_PFTO_WRAPPER(data.begin()));
+                                base64_binary ti_begin = base64_binary(data.begin());
                                 std::size_t padding = 2 - count % 3;
                                 while(--count > 0)
                                 {
